Pages/exportDataPage.py

```diff
@@ -228,7 +228,8 @@ def sms_exports(self):
         time.sleep(3)
         newest_file = latest_download_file()
         print("Newest:", newest_file)
-        modTimesinceEpoc = os.path.getmtime(str(UserInputsData.download_path) + "\\" + newest_file)
+        file_that_was_downloaded = os.path.join(UserInputsData.download_path + os.sep, newest_file)
+        modTimesinceEpoc = os.path.getmtime(file_that_was_downloaded)
```

Comment:
`os.sep` is not needed here. `os.path.join` will add it automatically.

Even better, if `UserInputsData.download_path` is a `Path` object you can do
```suggestion
        modTimesinceEpoc = (UserInputsData.download_path / newest_file).stat().st_mtime
```

`Path` objects have many useful [methods and properties](https://docs.python.org/3/library/pathlib.html#methods-and-properties)

Reply:
I had tried incorporating these changes and it was failing and I was unable to fix it. Lemme try again

Reply:
> os.sep is not needed here. os.path.join will add it automatically.

This throws error `unsupported operand type(s) for +: 'WindowsPath' and 'str'`